codalab/lib/bundle_cli.py

```diff
@@ -942,7 +942,7 @@ def do_upload_command(self, args):
             sources = [path_util.normalize(path) for path in args.path]
 
             print >>self.stderr, "Preparing upload archive..."
-            upload_blob, filename, total_bytes, unpack_at_server =\
+            upload_blob, filename, total_bytes, unpack_at_server, simplify_at_server =\
```

Comment:
5-tuples don't seem great since they're more error prone...maybe consider using a dict?  I don't feel super strongly about it, but just wanted to bring up the issue.